homeassistant/components/rfxtrx/config_flow.py

```diff
@@ -397,8 +397,14 @@ async def async_step_setup_serial_manual_path(self, user_input=None):
 
     async def async_step_import(self, import_config=None):
         """Handle the initial step."""
-        await self.async_set_unique_id(DOMAIN)
-        self._abort_if_unique_id_configured()
+        entry = await self.async_set_unique_id(DOMAIN)
+        # In version 0.113 devices is not stored in entry.data, to be safe update data for all VERSION=1 entries
```

Comment:
This will make it impossible to roll back after upgrading to the new version. Is that required or can we update the entry data in `async_setup_entry` instead?

Reply:
Thinking about it, we could perhaps keep `VERSION=1`. In 0.113 (see below) we were only writing device config to the entry data. Since 0.114, we add the `devices` key. I think we can check if `devices` key is missing and if so, update entry data with `devices` key (if present in yaml, do from yaml otherwise create an empty dict for `devices`)

```
hass.async_create_task(
        hass.config_entries.flow.async_init(
            DOMAIN,
            context={"source": config_entries.SOURCE_IMPORT},
            data={
                CONF_HOST: config[DOMAIN].get(CONF_HOST),
                CONF_PORT: config[DOMAIN].get(CONF_PORT),
                CONF_DEVICE: config[DOMAIN].get(CONF_DEVICE),
                CONF_DEBUG: config[DOMAIN][CONF_DEBUG],
            },
        )
    )
```

Reply:
Sounds good!